api/core/indexing_runner.py

```diff
@@ -211,9 +211,9 @@ def indexing_estimate(
         tenant_id: str,
         extract_settings: list[ExtractSetting],
         tmp_processing_rule: dict,
-        doc_form: str = None,
+        doc_form: Optional[str] = None,
         doc_language: str = "English",
-        dataset_id: str = None,
+        dataset_id: Optional[str] = None,
```

Comment:
Those two parameters should not be None

Reply:
Seems that `dataset_id ` could be None here, as Line 231 of `indexing_runner.py` does check the value nullable for different situation.